src/diagnostics_manager.js

```diff
@@ -93,6 +93,29 @@ class Routine {
   }
 }
 
+/**
+   * Returns true if a |functionName| exists in the chrome.os.diagnostics API.
+   * Returns false otherwise.
+   * @param {!string} functionName
+   * @return { !boolean }
+   */
+function isSupported(functionName) {
+  return chrome.os && chrome.os.diagnostics &&
+      chrome.os.diagnostics[functionName] &&
+      (typeof chrome.os.diagnostics[functionName] === 'function');
+}
+
+/**
+   * Returns a meaningful error message if a |functionName| doesn't exist in
+   * the chrome.os.diagnostics API.
+   * @param {!string} functionName
+   * @return { !string }
+   */
+function getErrorMessage(functionName) {
+  return 'DPSL: chrome.os.diagnostics.' + functionName + '() is not found.' +
+    ' Consider updating Google Chrome.';
```

Comment:
I think it will be a good idea to introduce a new Error type, so it will be easier to handle it
https://stackoverflow.com/questions/1433558/handling-specific-errors-in-javascript-think-exceptions